relocation.bs

```diff
@@ -742,21 +742,40 @@ This signature was picked as it completes the C++ tripartite value system.
 The copy constructor creates a new instance from an lvalue, the move constructor 
 from an xvalue, and then the relocation constructor from a prvalue.
 
-Despite what its signature suggests, the constructor does not take a copy of the
-source object as parameter. In fact, the constructor benefits from a special 
-aliasing to ensure that the address of its parameter is the same as the source
-object that is being relocated (as if the parameter was captured by reference).
-It is fine for the relocation constructor to provide such aliasing as constructors
-are special functions, which we cannot take the address of.
+Note: a further benefit of this syntax is that it is currently ill-formed, and
+thus available for extension.
+
+A point of confusion may be that the syntax implies an infinite regression: the
```

Comment:
infinite recursion?

Reply:
yes, but actually "infinite regress" - I think I committed a malapropism between the two.